be/src/olap/single_replica_compaction.h

```diff
@@ -30,20 +30,15 @@ class DataDir;
 //  SingleReplicaCompaction is used to fetch peer replica compaction result.
 class SingleReplicaCompaction final : public CompactionMixin {
 public:
-    SingleReplicaCompaction(StorageEngine& engine, const TabletSharedPtr& tablet,
-                            CompactionType compaction_type);
```

Comment:
I think it's not reasonable to remove compaction_type.

Reply:
I have rolled it back.

Reply:
The function of compact_type is to record whether the tablet should undergo base or cumulative compaction.